test/utils/disposable_test.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:async';
+
+import 'package:rxdart_ext/utils.dart';
+import 'package:test/test.dart';
+
+class _MockClass with Disposable {
+  _MockClass(
+    this.completer,
+  ) {
+    dispose$.listen((_) => completer.complete());
+  }
+
+  final Completer<void> completer;
+}
+
+void main() {
```

Comment:
Please include this test in `rxdart_ext_test.dart` file

Reply:
Done